docs/installation/basic_installation.md

```diff
@@ -58,7 +59,37 @@ If you would prefer less automation, or you  already have existing services on y
 
     <p>Once the preparations are done, the installer will prompt you to run <code>seat install:production</code>. </p>
 
-    <p>To configure ESI for SSO and API pulls, please refer to the <a href="/configuration/esi_configuration/">Configuring ESI</a> documentation page</p>
+</div>
+</section>
+
+!!! success "Installation complete"
+
+    You now have SeAT installed.
+    
+## Next steps
+
+1. [Configure ESI](/configuration/esi_configuration/) for SSO and API pulls
+2. Create a SeAT account with a character of yours
+3. log off or use new browser
+4. create admin link
+
+<section class="mdc-tabs">
```

Comment:
Please remove the tabs here.

Reply:
I am having trouble understanding and complying to this request. Prior we have stated, that for 1 line differences the tabs are the way to go. In this page we show 2 possible ways to install SeAT via Docker or the SeAT tool. Assuming that user chose one of the two ways this will result in two ways the are going to create the admin link. This is why the two options are listed. 
Further assuming that most user are not familiar with docker commands it is a far stretched assumption that they will go to the docker_admin page and figure the command out them-self. 

IMHO: If not here, we don't need tabs anywhere:
![image](https://user-images.githubusercontent.com/6583519/43364451-cbc1da32-931a-11e8-819f-a812c8e5ba02.png)
and:
![image](https://user-images.githubusercontent.com/6583519/43364456-d700f662-931a-11e8-937e-48262b8d0f4f.png)